CollAction/CollAction.csproj

```diff
@@ -1,14 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
-    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
-    <DisableImplicitFrameworkReference>true</DisableImplicitFrameworkReference>
     <UserSecretsId>aspnet-CollAction-29a5d7b8-3530-4482-9ed6-572bd6c178fa</UserSecretsId>
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
     <TypeScriptToolsVersion>Latest</TypeScriptToolsVersion>
-    <PreBuildEvent>npm install &amp;&amp; npm run build</PreBuildEvent>
+    <PreBuildEvent>npm install &amp;&amp; ^
+npm install --save bluebird &amp;&amp; ^
```

Comment:
You can you do "npm install --save bluebird" on your command prompt and commit the package.json and package.lock files. Build only needs 'npm install && npm run build' after that.

Reply:
But if the package.json changes again, due to dependency changes, you loose the bluebird change again.

This saves some setup time. Might be worth a look later if something better can be done about this.